wlm-ui/src/App.tsx

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
-import logo from './logo.svg';
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
 import './App.css';
+import InitPage from './InitPage/InitPage';
+import MainPage from './MainPage/MainPage';
+import { selectUser } from './store/slices/user/user';
 
 function App() {
+  const userState = useSelector(selectUser);
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      {userState.user === null ? (
+        <BrowserRouter>
+          <Routes>
+            <Route path='/init' element={<InitPage />} />
+            <Route path='*' element={<Navigate to='/init' replace />} />
+          </Routes>
+        </BrowserRouter>
+      ) : (
+        <BrowserRouter>
+          <Routes>
+            <Route path='/' element={<MainPage />} />
+            <Route path='*' element={<Navigate to='/' replace />} />
+          </Routes>
+        </BrowserRouter>
+      )}
```

Comment:
It seems the only difference is the path "/init" vs. "/". Is it the preferred option to do this way since more difference can be made afterwards?
Maybe just define a variable `const page = userState.use === null ? '/init' : '/'` would be better, if it is not the case.

Reply:
Sorry for the lack of explanation for my future plans.
I think there will be more pages such as a separate viewer of a certain channel and a PID viewer.
Hence, to ensure scalability, it looks better to keep as it is.